docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -268,51 +264,54 @@ endif::[]
 [role="_abstract"]
 A *sink* connector consumes messages from a Kafka topic and sends them to an external system.
 
-For this example, you use the *HTTP Sink* connector which consumes the Kafka messages (produced by the source {connectors} instance) and sends the messages to an HTTP endpoint.
+For this example, you use the *HTTP Sink* connector which consumes the Kafka messages (produced by your Data Generator source {connectors} instance) and sends the messages to an HTTP endpoint.
 
-ifndef::qs[]
 .Prerequisites
+
+ifndef::qs[]
 * You're logged in to the {product-long-connectors} web console at {service-url-connectors}[^].
+endif::[]
 * You created the source {connectors} instance as described in _Creating a {connectors} instance for a data source_.
 * For the data sink example, open the free https://webhook.site[webhook.site^] in a browser window. The `webhook.site` page provides a unique URL that you copy for use as an HTTP data sink.
-endif::[]
+
 
 .Procedure
 
 . In the {product-long-connectors} web console, click *Create {connectors} instance*.
 
 . Select the sink connector that you want to use:
-.. For example, type *http* in the search field. The list of {connectors} filters to show the *HTTP Sink* connector.
-.. Click the *HTTP Sink connector* card and then click *Next*.
+.. For example, type *http* in the search field. The list of {connectors} filters to show the *HTTP sink* connector.
+.. Click the *HTTP sink* card and then click *Next*.
 
-. Select the {product-kafka} instance for the connector to work with.
+. On the *Kafka instance* page, select the {product-kafka} instance for the connector to work with.
 +
 For example, select *test* and then click *Next*.
 
 . On the *Namespace* page, the namespace that you select depends on your {osd-name-short} environment. The namespace is the deployment space that hosts your {connectors} instances.
 +
 If you're using a trial cluster on your own {osd-name-short} environment, select the card for the namespace that was created when you added the {connectors} service to your trial cluster.
 +
-If you're using the evaluation {osd-name-short} environment, click the *eval namespace* that you created when you created the source connector.
+If you're using the hosted preview environment, click the *preview namespace* that you provisioned when you created the source connector.
```

Comment:
Similar comment as before: If the subsequent steps don't apply to both types of environment, clarify which environment they apply to.

Reply:
NPTF - the subsequent steps apply to both environments